libs/shared-atomic-design-components/src/lib/atoms/SelectStyles.tsx

```diff
@@ -0,0 +1,58 @@
+export const customStyles = {
```

Comment:
Good refactoring enabling these styles to be used two places.

The file name implies a component named `<SelectStyles>`.

Can we rename the file to maybe `FormSelect.styles.tsx`? Also, since the variable name `customStyles` now has less context (it's no longer inside of the `<FormSelect>` component), it would be nice to make the name clearer. What would be a better name?

Reply:
Maybe `SelectDropdown.styles.tsx` (since it's used not only in a form)? And `selectDropdownStyles` for the variable name?

Reply:
I would agree, but the problem is that it implies a component named `SelectDropdown`.

Maybe the other solution in my head can work?
- folder named `FormSelect` & move the `FormSelect`-related code into the folder
- put this file in there and just name it styles.ts
- actually create a `SelectDropdown` component in there, and use that in CON

Especially this last point makes things overall cleaner, by having clearer boundaries of separation between the modules. Instead of letting code in `apps/redi-connect` import both a component + its styles, it could import two variations of a component for two different use cases

Reply:
As we discussed last time, for clarity and following the best practices, I created a SelectDropdown component and reused `FormSelect.styles.tsx` to style it.